viewdir/login.njk

```diff
@@ -1,13 +1,46 @@
-{% block content%}
-{% if error %}
-<h3>{{ error }}</h3>
-{% endif %}
-<form action="/login" method="POST">
-  <label for="email">Email:</label><br>
-  <input type="text" id="email" name="email"><br>
-  <label for="password">Password:</label><br>
-  <input type="password" id="password" name="password"><br><br>
-  <input type="submit" value="Submit">
-</form>
+<head>
```

Comment:
This file is not extending the layout, so where are its `doctype`, `html` and `body` tags?